golang/internal/grpc/grpc.go

```diff
@@ -289,6 +309,33 @@ func grpcLoop(
 		command := new(agent.AgentCommand)
 		err = stream.RecvMsg(command)
 		if err != nil {
+			s := status.Convert(err)
+			if s != nil && (s.Code() == codes.Unauthenticated || s.Code() == codes.PermissionDenied || s.Code() == codes.NotFound) {
+				if l.AppConfig.GrpcToken.Type == config.Connection {
+					log.Error().Err(err).Msg("Invalid connection token. Removing")
```

Comment:
Small capital.

Reply:
As mentioned above, that is true for error strings only.